### PR TITLE
crvALUSD Deployment

### PR DESCRIPTION
**Pull request by smmackey**
Tranche details:
 - ~4.5 month term, ends on the same day as the most recent deployments
 - 10% initial APR
 - T-stretch: 11.10

### CODE REVIEW

**Review by jrhea — APPROVED** [review]
lgtm

**Review by DannyDelott — APPROVED** [review]
LGTM will integrate on FE once this lands

**Pull request merged**
